astropy/constants/__init__.py

```diff
@@ -68,10 +70,10 @@ def set_enabled_constants(modname):
     import warnings
     from . import utils as _utils
 
-    # NOTE: Update this when default changes.
+    # NOTE: Update this whenever versions are added.
```

Comment:
We could avoid having to remember to update by doing something along the lines of what you do in `config.py`, say
```
try:
    codata = importlib.importmodule('.' + modname + '.codata', 'astropy')
    iaudata = importlib.importmodule('.' + modname + '.iaudata', 'astropy')
except ImportError as exc:
    exc.args += ('Context manager does not currently handle {}'.format(modname),)
    raise
```

Reply:
Great idea! I've implemented it, slightly differently since it e.g. `astropy.constants.astropyconst13.codata` is not recognized as a module. But `astropy.constants.astropyconst13` is.